CONTRIBUTING.md

```diff
@@ -1,91 +1,214 @@
 		},
-		&utils.JSONReader{
-			FilePath: "salesforceCred2.json", // This reads from a file
-			JSONPath: "$.providerApp.clientSecret",
-			CredKey:  utils.ClientSecret,
+		Support: Support{
+			BulkWrite: BulkWriteSupport{
+             Insert: false,
+             Update: false,
+             Upsert: false,
+             Delete: false,
+            },
+			Proxy:     false,
+			Read:      false,
+			Subscribe: false,
+			Write:     false,
 		},
-       
-	}
-	credentialsRegistry.AddReaders(readers...)
-    credentialsRegistry.AddReader(
-        &ValueReader{
-            Val:     "some-salesforce-instance-subdomain", // Hard coded value
-            CredKey: "workspaceRef",
-        },
-    )
-	salesforceWorkspace := credentialsRegistry.MustString("workspaceRef")
-
-	cfg := utils.SalesforceOAuthConfigFromRegistry(credentialsRegistry)
-	tok := utils.SalesforceOauthTokenFromRegistry(credentialsRegistry)
-	ctx := context.Background()
-
-	sfc, err := connectors.Salesforce(
-		salesforce.WithClient(ctx, http.DefaultClient, cfg, tok),
-		salesforce.WithWorkspace(salesforceWorkspace))
-	if err != nil {
-		slog.Error("Error creating Salesforce connector", "error", err)
-
-		return
-	}
-
-```
\ No newline at end of file
+    },
+
+    // API key provider
+    SixSense: {
+        AuthType: ApiKey,
+        BaseURL: "https://api.6sense.com",
+        // For 6sense, the header needs to be 'Authorization: Token {your_api_key}'
+        ApiKeyOpts: &ApiKeyOpts{
+            Type:        InHeader, // Can also be InQuery
+			HeaderName: "Authorization",
+            ValuePrefix: "Token ",
+            DocsURL: "https://api.6sense.com/docs/#get-your-api-token",
+		},
+        // For another provider, ValuePrefix may not be needed
+        // For example, if the expected header is 'X-Api-Key: {your_api_key}'
+        /*
+        ApiKeyOpts: &ApiKeyOpts{
+			HeaderName: "X-Api-Key",
+            DocsURL: "https://api.6sense.com/docs/#get-your-api-token",
+		}, */
+		Support: Support{
+			BulkWrite: BulkWriteSupport{
+             Insert: false,
+             Update: false,
+             Upsert: false,
+             Delete: false,
+            },
+			Proxy:     false,
+			Read:      false,
+			Subscribe: false,
+			Write:     false,
+		},
+    },
+
+    // Basic auth provider
+    Insightly: {
+        AuthType: Basic,
+        BaseURL: "https://api.{{.pod}}.insightly.com",
+		Support: Support {
+			BulkWrite: BulkWriteSupport{
+             Insert: false,
+             Update: false,
+             Upsert: false,
+             Delete: false,
+            },
+			Proxy:     false,
+			Read:      false,
+			Subscribe: false,
+			Write:     false,
+		},
+        PostAuthInfoNeeded: true,
+    },
+```
+
+
+### Testing your proxy connector
+
+Proxy server script
+
+`go run scripts/proxy/proxy.go` 
+
+This will run the proxy server at port `:4444` 
+
+You can now make requests against `localhost:4444` and they will be routed to the provider you're working on.
```

Comment:
```suggestion
This will run the proxy server at port `:4444`. You can now make requests against `localhost:4444` and they will be routed to the provider you're working on.
```